scripts/check-go-vulnerabiltiies.sh

```diff
@@ -0,0 +1,33 @@
+
+#!/usr/bin/env bash
+
+# This script checks for go vulnerabilities.
+
+govul="govulncheck"
+
+# Test if govulncheck is installed as a binary or a go module.
+if command -v govul &> /dev/null; then
+    # If the user has installed swaggo as a binary, use the binary.
+    swaggo=$(command -v govul)
```

Comment:
Is the `swaggo` here from a copy/paste maybe?

Reply:
Yup should be switched to govul!